langgraph/src/pregel/write.ts

```diff
@@ -92,6 +93,15 @@ export class ChannelWrite<
       Object.entries(values).filter(([_channel, value]) => value !== SKIP_WRITE)
     );
   }
+
+  static isWriter(runnable: RunnableLike): boolean {
+    // eslint-disable-next-line no-instanceof/no-instanceof
+    return runnable instanceof ChannelWrite;
+  }
+
+  static registerWriter(runnable: Runnable): ChannelWrite {
+    return runnable as ChannelWrite;
```

Comment:
Double check: Not used at the moment, but need to double check that this is the desired implementation.

Reply:
It's not, but you can fix it later